src/Dapr.AspNetCore/DaprAuthenticationBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    using System;
+    using Dapr;
+    using Dapr.AspNetCore;
```

Comment:
Outside the `namespace` please. That's the preferred style and the repo still has a mix unfortunately.

Reply:
Fixed